Makefile

```diff
@@ -32,6 +32,18 @@ run-end-to-end-test:
 	docker-compose run run-tests; STATUS=$$?; \
 	docker-compose down; exit $$STATUS
 
+# Stop and clean running environment.
+
+stop-experiment: export COMPOSE_PROJECT_NAME := fuzzbench
```

Comment:
Can you make this one target please?

Reply:
I guess different COMPOSE_PROJECT_NAME needs different targets?

Reply:
Done.